imports/ui/Results.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import "./css/Results.css"; // css files
+import Navbar from './Navbar.jsx';
+import ResultsDisplay from './ResultsDisplay.jsx';
+import PropTypes from "prop-types";
+
+
+/*
+  Results Component. Short description if needed.
+
+  Identify as one of the following components:
+  Simple styling: mainly renders HTML and CSS,
+  Container: combines multiple components into a single feature
+  View: top-level component accessed by a URL endpoint defined by the router in main.jsx
+
+  Include a breif description of the component's purpose, where it falls in the
+  component tree, and any inportant information it accesses or modifies.
+  Include the route for View components.
+*/
```

Comment:
The docs look useless

Reply:
This is a template that was never changed, we'll change it before merging.

Reply:
Fixed in latest commit.